### ISSUE TITLE
[Quest] Planting the Banner

### ISSUE THREAD

**Issue by Druhl** [WoWManiaUK/Redemption#2792]
(http://www.wow-mania.com/armory/?quest=10518)

After planting banner, no mobs came up ramp. Flew back to quest giver, abandoned and re-took quest, planted banner, still no mobs came up ramp.

Flew around tower to look and saw all had spawned at foot of ramp and were "stuck". Also from the screenshots you can see the spawns from my first attempt were still there (you can see there are 2 of the "final" mob Gurn there).

I was able to complete the quest, but just FYI.

![wowscrnshot_101218_015050](https://user-images.githubusercontent.com/19803911/46884703-c50de700-ce23-11e8-909b-ef90703621d4.jpg)
![wowscrnshot_101218_015215](https://user-images.githubusercontent.com/19803911/46884708-c93a0480-ce23-11e8-914a-d83cfa48c17e.jpg)

**Comment by wow-mania** [comment]
Can someone confirm this on PTR. I am pretty sure it works.

**Comment by Lucie93** [comment]
I'm on in atm.

**Comment by Lucie93** [comment]
Ok so... Only thing is that those mobs won't go up on the tower and instead they all stack below it. 
It takes a bit of time till the ''boss'' mob appears but thats cause it's ''hard'' lvl 70 quest for 3 people and they give you time to heal up etc. till the harder mob spawns.

**Comment by Druhl** [comment]
You're right they are intended to spawn in "waves" with a delay between, but they are supposed to come up the ramp.

Also 2nd issue that whole second set of spawns appeared after I abandoned and re-started quest.

**Comment by Lucie93** [comment]
I agree that they don't come up the tower and spawn below and stay there.
But otherwise it worked fine for me. Everything spawned after pause. No extra or missing mobs.
